### PR TITLE
Adding Program to test movement

### PR DESCRIPTION
**Pull request by PhobosLegacy**

### CODE REVIEW

File: SerialPrograms/Source/CommonFramework/Options/EditableTableOption.cpp [hunk 1]
```diff
@@ -90,6 +90,34 @@ template<typename T> QWidget* make_integer_table_cell(QWidget& parent, T& value)
     return box;
 }
 
+
+QWidget* make_limited_integer_table_cell(QWidget& parent, uint8_t& value, uint8_t min, uint8_t max){
```
Comment (Mysticial) [inline SerialPrograms/Source/CommonFramework/Options/EditableTableOption.cpp:94]: Do we need to limit this to only 8-bit integers?

Comment (PhobosLegacy) [reply SerialPrograms/Source/CommonFramework/Options/EditableTableOption.cpp]: changed to int :)

File: SerialPrograms/Source/CommonFramework/Options/EditableTableOption.cpp [hunk 2]
```diff
@@ -90,6 +90,34 @@ template<typename T> QWidget* make_integer_table_cell(QWidget& parent, T& value)
     return box;
 }
 
+
+QWidget* make_limited_integer_table_cell(QWidget& parent, int& value, int min, int max){
```
Comment (Mysticial) [inline SerialPrograms/Source/CommonFramework/Options/EditableTableOption.cpp:94]: Can we actually change this to either `int32_t` or `int64_t`. The size of `int` is not defined for C++.

Comment (PhobosLegacy) [reply SerialPrograms/Source/CommonFramework/Options/EditableTableOption.cpp]: done!

**Pull request merged**